lib/ext/python_libstorj.cpp

```diff
@@ -64,6 +64,37 @@ void delete_bucket_cb(uv_work_t *work_req, int status) {
     PyObject_CallFunction(_handle, "s", error_str);
 }
 
+void get_bucket_id_cb(uv_work_t *work_req, int status) {
+    char *error_str = (char *)calloc(255, sizeof(char));
+    PyObject *error = Py_None;
+    get_bucket_id_request_t *req = (get_bucket_id_request_t *)work_req->data;
+    PyObject *_handle = (PyObject *)req->handle;
+
+    PyObject *bucket_dict = PyDict_New();
+    if (error_and_status_check<get_bucket_id_request_t>(req, &error_str)) {
+        PyDict_SetItemString(bucket_dict, "name", PyString_FromString(req->bucket_name));
+        PyDict_SetItemString(bucket_dict, "id", PyString_FromString(req->bucket_id));
+        // TODO: manage encrypted bucket name
```

Comment:
@marcorosa what did you have in mind here?

For reference:
- [storj.c:184](https://github.com/Storj/libstorj/blob/893a729579c8e68afd844d2fc949a4e786922d87/src/storj.c#L184) is where libstorj is encrypting the bucket name (before making a request to the [`/bucket-ids/` route of bridge](https://github.com/Storj/bridge/blob/master/lib/server/routes/buckets.js#L123) at [storj.c:203](https://github.com/Storj/libstorj/blob/893a729579c8e68afd844d2fc949a4e786922d87/src/storj.c#L203) using the path from [storj.c:197](https://github.com/Storj/libstorj/blob/893a729579c8e68afd844d2fc949a4e786922d87/src/storj.c#L197)).
- [crypto.c:185](https://github.com/Storj/libstorj/blob/23d5c55e7e0a0564dcd4c9a9199bf0612279f025/src/crypto.c#L185) is where `encrypt_bucket_name` is defined.